jazzband/members/views.py

```diff
@@ -21,8 +21,11 @@ def roadies():
 
 @members.route("/roadies/issue")
 def roadies_issue():
-    labels = request.args.get("labels", None)
-    url = "https://github.com/jazzband-roadies/help/issues/new"
-    if labels:
-        url += f"?labels={labels}"
+    url = "https://github.com/jazzband-roadies/help/issues/new/choose"
+    return redirect(url)
+
+
+@members.route("/website/issue")
+def website_issue():
+    url = "https://github.com/jazzband-roadies/website/issues/new"
```

Comment:
Actually let's remove that, I want to keep all issue coming from the website in the help repo, just added another template for the website: https://github.com/jazzband-roadies/help/issues/new?assignees=&labels=bug&template=website.md&title=Bug+found+on+website